templates/CRM/Import/Form/MapTableCommon.tpl

```diff
@@ -68,17 +68,24 @@
             } else {
               cj('#saveDetails').hide();
             }
-
-            function showSaveDetails(chkbox) {
```

Comment:
@eileenmcnaughton I think this breaks https://github.com/civicrm/civicrm-core/blob/a1c3dbf315a100d0924d3043d937162852290e24/CRM/Import/Form/MapField.php#L159 as won't it mean there is an undefined function showSaveDetails in the console log?

Reply:
@seamuslee001 thanks - yeah I think that can go - I didn't notice breakage but might have missed it

Reply:
@seamuslee001 OK - I removed it & it seems to be working fine